src/apis/quipuswap-staking/index.ts

```diff
@@ -0,0 +1,135 @@
+import { MichelsonMap, TezosToolkit } from '@taquito/taquito';
+import axios from 'axios';
+import { BigNumber } from 'bignumber.js';
+
+import { toIntegerSeconds } from 'src/utils/date.utils';
+import { getFirstAccountActivityTime } from 'src/utils/earn.utils';
+import { READ_ONLY_SIGNER_PUBLIC_KEY_HASH } from 'src/utils/env.utils';
+import { isDefined } from 'src/utils/is-defined';
+import { getReadOnlyContract } from 'src/utils/rpc/contract.utils';
+import { parseTransferParamsToParamsWithKind } from 'src/utils/transfer-params.utils';
+
+import { calculateStableswapLpTokenOutput, calculateStableswapWithdrawTokenOutput } from './stableswap-calculations';
+import { FarmsListResponse, StableswapPoolStorage, TooLowPoolReservesError } from './types';
+
+const stakingApi = axios.create({ baseURL: 'https://staking-api-mainnet.prod.quipuswap.com' });
```

Comment:
It might make sense to move this to a separate environment variable rather than hardcode the URL (like with other API URLs across the project)